website/content/docs/upgrading/upgrade-to-1.10.x.mdx

```diff
@@ -62,17 +62,35 @@ OIDC provider system to reduce configuration steps and enhance usability.
 The following built-in resources are included in each Vault namespace starting with Vault
 1.10:
 
- - A "default" OIDC provider that's usable by all client applications
- - A "default" key for signing and verification of ID tokens
- - An "allow_all" assignment which authorizes all Vault entities to authenticate via a
-   client application
+- A "default" OIDC provider that's usable by all client applications
+- A "default" key for signing and verification of ID tokens
+- An "allow_all" assignment which authorizes all Vault entities to authenticate via a
+  client application
 
 If you created an [OIDC provider](/api-docs/secret/identity/oidc-provider#create-or-update-a-provider)
 with the name "default", [key](/api-docs/secret/identity/tokens#create-a-named-key) with the
 name "default", or [assignment](/api-docs/secret/identity/oidc-provider#create-or-update-an-assignment)
-with the name "allow_all" using the Vault 1.9 tech preview, the installation of these built-in
-resources will be skipped. We _strongly_ recommend that you delete any resources that have
+with the name "allow*all" using the Vault 1.9 tech preview, the installation of these built-in
```

Comment:
This isn't formatting correctly but I'm not sure exactly what you need to change.

Reply:
I'm not sure either. I may have to do consult with other writers and see if there's a way to reformat it correctly.

Reply:
I'm not 100% sure what you were going for but

```
the name "allow_all" using the Vault 1.9 tech preview, the installation of these built-in
resources will be skipped. We *strongly*
```

seemed to work:
<img width="940" alt="Screen Shot 2022-03-23 at 17 48 54" src="https://user-images.githubusercontent.com/9422872/159801742-7ac74266-657f-402d-bddd-b432e9072609.png">

Reply:
```
the name "allow\_all" using the Vault 1.9 tech preview, the installation of these built-in
resources will be skipped. We _strongly_
```

Also works if you escape the underscore in "allow_all" in the bullet list above, I think.